README.md

```diff
@@ -110,7 +110,7 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? =>  moon goodnight
+- What is mystery6("Ada Developers Academy")? => Academy Developers Ada
+- What is mystery6("Hi, there!")? => there! Hi,
```

Comment:
This looks good, but you should always use double quotes around strings. It's particularly important in this case because there is a leading space on each of these answers that we can't see without the qutations.

So it's " moon goodnight" not "moon goodnight" but I can't tell if you knew that or not because of the lack of quotations